cicerone/app/src/main/java/com/teampower/cicerone/database/CiceroneAppDatabase.kt

```diff
@@ -0,0 +1,83 @@
+package com.teampower.cicerone.database
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import androidx.sqlite.db.SupportSQLiteDatabase
+import com.teampower.cicerone.database.category_table.CategoryDao
+import com.teampower.cicerone.database.history_table.POIHistoryDao
+import com.teampower.cicerone.database.history_table.POISavedDao
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+@Database(entities = [POIHistoryData::class, POISavedData::class, CategoryData::class], version = 1)
+public abstract class CiceroneAppDatabase : RoomDatabase() {
+
+    // abstract fun POIDao(): POIDao
+    abstract fun poiHistoryDao(): POIHistoryDao
+    abstract fun poiSavedDao(): POISavedDao
+    abstract fun categoryDao(): CategoryDao
+
+    private class CiceroneDatabaseCallback(
+        private val scope: CoroutineScope
+    ) : RoomDatabase.Callback() {
+
+        override fun onOpen(db: SupportSQLiteDatabase) {
+            super.onOpen(db)
+            INSTANCE?.let { database ->
+                scope.launch {
+                    //************* POI History table *************
+                    val poiHDao = database.poiHistoryDao()
+                    // Delete all content here.
+                    poiHDao.deleteAll()
+                    // Add sample POI.
+                    val poiH = POIHistoryData("1337XD", "SNU", "School", "12:00", 0.00, 0.00)
```

Comment:
This is just test data so it's understandable, but just a note that the time triggered would be a proper datetime string, not only the clock "12:00". Just so we know what we can expect from the database.